app/services/analytics_service.rb

```diff
@@ -1,20 +1,10 @@
 class AnalyticsService
```

Comment:
Renaming it to `Analytics::Pro` makes sense. I think it's fine to have it in a separate PR, too.